changelogs/unreleased/gh-9806-http-set-version.md

```diff
@@ -0,0 +1,3 @@
+## feature/lua/http client
```

Comment:
Please, add a documentation request into the commit message (see examples in other commits).

Reply:
Documentation request added

Reply:
Thanks!

Minor nits:

* I would mention that the option also appears in all the methods specialized to a request type: `:get()`, `:post()` and so on.
* I would lease a link to the curl's option.

Reply:
Fixed. Added a link